app/src/main/java/io/homeassistant/companion/android/matter/MatterCommissioningRequest.kt

```diff
@@ -0,0 +1,11 @@
+package io.homeassistant.companion.android.matter
+
+object MatterCommissioningRequest {
+
+    enum class Status {
```

Comment:
Why are we wrapping this enum in an object?

Reply:
Because originally there was more in the object, and I forgot about this when cleaning it up. I'll remove the object and rename the enum.